plugins/PermissionsEx/permissions.yml

```diff
@@ -113,6 +113,7 @@ groups:
     - permissions.user.promote.default
     - permissions.user.rank.demote
     - permissions.user.rank.promote
+    - logblock.lookup
```

Comment:
You can push more commits to the branch and they'll automatically be reflected here.